src/store/getters/state-getters.ts

```diff
@@ -154,7 +153,8 @@ export const isBusySaving = (state: any): boolean => {
  * Whether all the incorporation steps are valid.
  */
 export const isApplicationValid = (state: any): boolean => {
-  return (state.stateModel.defineCompanyStep.valid && state.stateModel.addPeopleAndRoleStep.valid)
+  return (state.stateModel.defineCompanyStep.valid && state.stateModel.addPeopleAndRoleStep.valid &&
+    state.stateModel.createShareStructureStep.valid && state.stateModel.incorporationDateTime.valid)
```

Comment:
Should this also look at certifyState?

Reply:
I'll look into this!

Reply:
Good question, this directly correlates to the stepper appearance of the green Checkmark. Do we want that checkmark to appear when all previous steps are valid or only once all of Step 5 is complete as well and the file and pay is ready? 

This means the user will need enter their name and give a certify Check, which will then result in the stepper appearance of the green Checkmark. 

Can totally update this tho, thoughts on this?

Reply:
I think the step 5 page forms fields should affect the step 5 green checkmark, whereas the overall application state should affect the File + Pay button.
However, Scott may have a different experience in mind... ask/confirm with him.

PS - Before Ody's validation change, step 5 page would get recreated each time you landed on it, and the Certify would get cleared -- which I didn't like (IMO). I assume that now, once I have checked Certify, it stays checked during this session (and only gets cleared when I resume)?

Reply:
K, will follow up with Scott on this interaction!

I'm not 100% sure. I will rebase in Ody's work and verify this 👍

Reply:
rebased and ody's work is excellent, really helps! Things are looking good. 
The certify is cleared once resuming, correct 👍 
Also, enquired with Scott about when we want to present the validity on the stepper, will update when i find out!